src/easylink/step.py

```diff
@@ -1262,25 +1287,21 @@ def validate_step(
                     f"'{step_config.type}' is not a supported 'type'. Valid choices are: {list(self.choices)}."
                 ]
             }
-        # Handle type-subgraph inconsistencies
-        subgraph = self.choices[chosen_type]
+
+        chosen_step = self.choices[chosen_type]["step"]
         chosen_step_config = LayeredConfigTree(
             {key: value for key, value in step_config.items() if key != "type"}
         )
-        allowable_steps = [node.name for node in subgraph["nodes"]]
-        if set(allowable_steps) != set(chosen_step_config):
+        if chosen_step.name not in chosen_step_config:
             return {
                 f"step {self.name}": [
-                    f"Invalid configuration for '{chosen_type}' type. Valid steps are {allowable_steps}."
+                    f"'{chosen_step.name}' is not configured. Confirm you have specified "
```

Comment:
Removing the check that the provided steps match the expected for a given `type` did NOT raise as I thought it would; it raised a KeyError b/c it was basically looking for the wrong key. This seems like a compromise (and exactly what we do [here](https://github.com/ihmeuw/easylink/blob/sbachmei/mic-5897/update-choice-step/src/easylink/step.py#L657-L658) as well).

Reply:
👍  I expect this to go away soon-ish anyway, since we will want `ChoiceStep` to be one of the places where the child's step name is elided.